src/main/atm2lndType.F90

```diff
@@ -553,6 +555,10 @@ subroutine InitHistory(this, bounds)
          avgflag='A', long_name='atmospheric surface height', &
          ptr_lnd=this%forc_topo_grc)
 
+    call hist_addfld1d (fname='ATM_O3', units='mol/mol', &
+         avgflag='A', long_name='atmospheric ozone partial pressure', &
+         ptr_lnd=this%forc_o3_grc)
```

Comment:
- [ ] I think this should be inactive by default but then added to the ozone testmod(s)

Reply:
I just added this to the o3 test [def variable list ](https://github.com/adrifoster/CTSM/blob/ozone_update/cime_config/testdefs/testmods_dirs/clm/o3lombardozzi2015/user_nl_clm)